translations/catroweb.en.yml

```diff
@@ -184,6 +184,8 @@ programs:
   link: Copy project link to clipboard
   reactionsText: Reactions
   downloadErrorText: Error occurred while downloading the project. Please try again later.
+  provider: Translated by %provider% from
+  creditTransition: to
```

Comment:
I think it might be better for this to be one string resource. It will be easier for human translators to translate "to" to the correct word in that language. "to" in English can mean a lot of things that might be different words in other languages. I know this might be hard in the future to add language selectors in here...

Reply:
Converged into one resource and added logic to handle splitting it up into the different containers. Also created logic to handle if the source language is after target language